### ISSUE TITLE
Migrate Projects to .NET Core 3.0

### ISSUE THREAD

**Issue by pcfulife** [unosquare/ffmediaelement#307]
# .NET Core 3.0 compatibility issue

*With Visual Studio 2019 and .NET Core 3.0 Preview 1, FFME.Windows Nuget package can be installed but cannot be referenced*

## Issue Categories

 - [x] Bug
 - [ ] Feature Request
 - [ ] Question
 - [ ] Not sure

## Version Information

 - [x] NuGet Package *4.0.270*
 - [ ] Build From *Master* branch, Commit *Enter commit id. Example: 704c482*
 - [ ] Build from *Branch Name*, Commit *Enter commit id. Example: 704c482*

## Steps to Reproduce

 1. Install Visual Studio 2019 and .NET Core 3.0 Preview 1.
 2. Type `dotnet new wpf -o WpfApp1`.
 3. In Visual Studio, create a empty solution.
 4. Load WpfApp1 project into solution.
 5. Install FFME.Windows Nuget package.

## Expected Results

 - Compiler cannot find ffme:MediaElement.
 - Unosquare namespace is not found.
 
### XAML
```xml
<MainWindow xmlns:ffme="clr-namespace:Unosquare.FFME;assembly=ffme.win">
    <ffme:MediaElement />
</MainWindow>
```

**Comment by mariodivece** [comment]
That's right. I would need to move from .NET 4.6.1 to Net Core 3.0. I will wait until netcore 3.0 is final before I magrate this project over.

**Comment by mariodivece** [comment]
Thanks for the PR. This gives me a very good starting point when 3.0 final comes out!

**Comment by mariodivece** [comment]
The projects are already updated to the latest preview. Still need to wait for final release of dotnet core 3.0

**Comment by mariodivece** [comment]
netcore 3.0 preview 3 will be available on 2.90 release. Closing.
